Tools/autotest/param_metadata/rstemit.py

```diff
@@ -205,20 +205,26 @@ def emit(self, g, f):
         for param in g.params:
             if not hasattr(param, 'DisplayName') or not hasattr(param, 'Description'):
                 continue
+                
+            # Check if frame_type defined
+            frame_type = '' if not hasattr(param, 'FrameType') else param.FrameType.strip()
```

Comment:
This seems... complicated.

Can this be done as `frame_type = param.FrameType.strip() if hasattr(param, 'FrameType') else ''`

Reply:
Sure. Done.